ReactAndroid/src/main/jni/react/jni/NativeTime.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <time.h>
+#include "NativeTime.h"
+
+namespace facebook {
+namespace react {
+
+double reactAndroidNativePerformanceNowHook() {
+  struct timespec now;
+  clock_gettime(CLOCK_MONOTONIC, &now);
```

Comment:
Any reason for not using std::steady_clock/high_resolution_clock ? They usually end up calling clock_gettime on POSIX based platforms .. That would allow this fix to apply for all engines/platforms (including windows !)

Reply:
Good point, thanks! I changed it to high_resolution_clock, I can confirm it still works on Android.

Reply:
The standard says that: "The high_resolution_clock is not implemented consistently across different standard library implementations, and its use should be avoided. It is often just an alias for std::chrono::steady_clock or std::chrono::system_clock, but which one it is depends on the library or configuration."
For performance measurements we need a monotonic clock so you should probably use steady_clock here 🙂

Reply:
Thanks!